### DISCUSSION TITLE
Rtlamr in Ubuntu core

### DISCUSSION THREAD

**Discussion by uriva9w** [bemasher/rtlamr#232] [Ideas]
Hi
It could be interesting to have RTLAMR running in embedded IOT devices e.g. Ubuntu core based devices
Any ideas or comments?

**Comment by bemasher** [comment]
I don't think there's any reason rtlamr couldn't run in Ubuntu core, but it does have some minimum hardware requirements. I don't think it will run on anything less than an RPi 2.

**Comment by camaz** [comment]
I had success on a raspberry pi zero w running alpine linux and the patched rtl-sdr with ring buffer from Stephen Blinick at https://github.com/blinick/rtl-sdr.
